src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -252,19 +252,19 @@
                 <div class="slds-col slds-size--1-of-2 slds-m-bottom--medium">
                     {!$Label.c.stgRefreshAdminAcctNameTitle}
                     <div class="slds-col slds-size--1-of-1">
-                    	<ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
+                        <ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
                     </div>
                 </div>
 
-              	<div class="slds-col slds-size--1-of-2">
-              		<lightning:button
+                  <div class="slds-col slds-size--1-of-2">
```

Comment:
Spacing not aligned here either.

Reply:
I'll take a look, will have to wait until tomorrow though.